.travis.yml

```diff
@@ -1,20 +1,26 @@
 language: c
-install: wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-docker.sh
+install: wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-docker.sh
 script: bash -ex .travis-docker.sh
 services:
   - docker
 sudo: false
 env:
  global:
-   - PACKAGE="publish"
+   - PACKAGE="opam-publish"
+   - EXTRA_ENV="OPAMSOLVERTIMEOUT=600"
    - PRE_INSTALL_HOOK="cd /home/opam/opam-repository && git pull origin master && opam update -u -y"
  matrix:
-   - DISTRO=debian-stable OCAML_VERSION=4.02.3
-   - DISTRO=debian-testing OCAML_VERSION=4.03.0
-   - DISTRO=debian-unstable OCAML_VERSION=4.04.0
-   - DISTRO=ubuntu-12.04 OCAML_VERSION=4.02.3
-   - DISTRO=ubuntu-16.04 OCAML_VERSION=4.04.0
-   - DISTRO=centos-6 OCAML_VERSION=4.04.0
-   - DISTRO=centos-7 OCAML_VERSION=4.03.0
-   - DISTRO=fedora-24 OCAML_VERSION=4.02.3
-   - DISTRO=alpine OCAML_VERSION=4.04.0
+   - DISTRO=debian-testing OCAML_VERSION=4.05
+   - DISTRO=debian-unstable OCAML_VERSION=4.05
+   - DISTRO=ubuntu-16.04 OCAML_VERSION=4.04
+   - DISTRO=ubuntu-18.04 OCAML_VERSION=4.05
+   - DISTRO=centos-8 OCAML_VERSION=4.05
```

Comment:
```suggestion
   - DISTRO=centos-7 OCAML_VERSION=4.05
```

Reply:
Not required anymore since your commit was merged in https://github.com/avsm/ocaml-dockerfile/commit/9bed88bbfd6346d98b448243b5b12a3329bfb56f

Reply:
I'd rather not wait a week or more again just to have this check (docker images are only updated once in a while)

Reply:
Well, fine to me, assuming it will not take waiting a week (or more) anyway :D

Reply:
We would be able to update it afterwards.